### PR TITLE
[CTOR-657] refresh Azure Cache for Redis doc page

### PR DESCRIPTION
**Pull request by YanMeddour**
## Description

Please include a short summary of the changes and what is the purpose of the PR. Any relevant information should be added to help reviewers.

## Target version (i.e. version that this PR changes)

- [ ] 22.10.x
- [ ] 23.04.x
- [ ] 23.10.x
- [ ] 24.04.x
- [ ] 24.10.x
- [ ] Cloud
- [ ] Monitoring Connectors

### CODE REVIEW

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-azure-database-redis.md [hunk 1]
```diff
@@ -5,288 +5,742 @@ title: Azure Cache for Redis
...
 import TabItem from '@theme/TabItem';
 
-
 ## Vue d'ensemble
 
 Azure Cache pour Redis fournit un magasin de données en mémoire basé sur le logiciel Redis. Redis améliore les performances et l’extensibilité d’une
 application qui utilise abondamment les magasins de données principaux. Il peut traiter des volumes conséquents de demandes d’applications en conservant
 les données fréquemment utilisées dans la mémoire du serveur, qui peut être écrite et lue rapidement. Redis constitue une solution de stockage de
-données à faible latence et à haut débit critique pour les applications modernes. 
+données à faible latence et à haut débit critique pour les applications modernes.
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-azure-database-redis.md:13]:
```suggestion
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-azure-database-redis.md [hunk 2]
```diff
@@ -5,288 +5,742 @@ title: Azure Cache for Redis
 import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 
-
 ## Vue d'ensemble
 
 Azure Cache pour Redis fournit un magasin de données en mémoire basé sur le logiciel Redis. Redis améliore les performances et l’extensibilité d’une
 application qui utilise abondamment les magasins de données principaux. Il peut traiter des volumes conséquents de demandes d’applications en conservant
 les données fréquemment utilisées dans la mémoire du serveur, qui peut être écrite et lue rapidement. Redis constitue une solution de stockage de
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-azure-database-redis.md:12]:
```suggestion
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-azure-database-redis.md [hunk 3]
```diff
@@ -5,288 +5,742 @@ title: Azure Cache for Redis
...
-données à faible latence et à haut débit critique pour les applications modernes. 
+données à faible latence et à haut débit critique pour les applications modernes.
 
-Le connecteur de supervision Centreon *Azure Cache for Redis* s'appuie sur les API Azure Monitor afin de récuperer les métriques relatives au service
+Le connecteur de supervision Centreon **Azure Cache for Redis** s'appuie sur les API Azure Monitor afin de récuperer les métriques relatives au service
 Cosmos DB. Il est possible d'utiliser les 2 modes proposés par Microsoft: RestAPI ou Azure CLI.
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-azure-database-redis.md:16]: Cosmos DB?

File: pp/integrations/plugin-packs/procedures/cloud-azure-database-redis.md [hunk 4]
```diff
@@ -5,7 +5,6 @@ title: Azure Cache for Redis
 import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 
-
 ## Overview
 
 Azure Cache for Redis provides an in-memory data store based on the Redis software. Redis improves the performance and scalability of an application
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/cloud-azure-database-redis.md:10]:
```suggestion
```

File: pp/integrations/plugin-packs/procedures/cloud-azure-database-redis.md [hunk 5]
```diff
@@ -14,270 +13,730 @@ memory, which can be written to and read from quickly. Redis brings a critical l
...
+| AZURERESOURCEGROUP | Set resource group (required if resource's name is used)                                                                   |                   | X           |
+| AZURESUBSCRIPTION  | Set Azure subscription (required if logged to several subscriptions)                                                       |                   | X           |
+| AZURETENANT        | Set Azure tenant ID                                                                                                        |                   | X           |
+| PROXYURL           | Proxy URL if any                                                                                                           |                   |             |
+| EXTRAOPTIONS       | Any extra option you may want to add to every command (a --verbose flag for example). All options are listed [here](#available-options).                       |                   |             |
+
+> Set the following options in the EXTRAOPTIONS macro if you are monitoring resource from Microsoft Azure operated by 21Vianet (Azure China):
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/cloud-azure-database-redis.md:257]:
```suggestion
> Set the following options in the EXTRAOPTIONS macro if you are monitoring resources from Microsoft Azure operated by 21Vianet (Azure China):
```

**Review by cg-tw — CHANGES_REQUESTED** [review]

**Review by lucie-dubrunfaut — APPROVED** [review]

**Review by cg-tw — APPROVED** [review]

**Pull request merged**